doc/source/user_guide/enhancingperf.rst

```diff
@@ -237,18 +240,18 @@ the rows, applying our ``integrate_f_typed``, and putting this in the zeros arra
 
 .. code-block:: ipython
 
-   In [4]: %timeit apply_integrate_f(df['a'].to_numpy(),
-                                     df['b'].to_numpy(),
-                                     df['N'].to_numpy())
+   In [4]: %timeit apply_integrate_f(
+                    df["a"].to_numpy(), df["b"].to_numpy(), df["N"].to_numpy()
+                )
```

Comment:
This might need to be a `%%timeit` with the whole block starting on the second line (at which point you probably don't need to wrap lines anymore):

```
[ins] In [7]: %%timeit sum(
         ...:     [1, 2, 3]
         ...: )
         ...:
         ...:
Traceback (most recent call last):

  File "/Users/danielsaxton/opt/miniconda3/envs/pandas-dev/lib/python3.8/site-packages/IPython/core/interactiveshell.py", line 3417, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)

  File "<ipython-input-7-a320ae5b655e>", line 1, in <module>
    get_ipython().run_cell_magic('timeit', 'sum(', '    [1, 2, 3]\n)\n\n')

  File "/Users/danielsaxton/opt/miniconda3/envs/pandas-dev/lib/python3.8/site-packages/IPython/core/interactiveshell.py", line 2381, in run_cell_magic
    result = fn(*args, **kwargs)

  File "<decorator-gen-53>", line 2, in timeit

  File "/Users/danielsaxton/opt/miniconda3/envs/pandas-dev/lib/python3.8/site-packages/IPython/core/magic.py", line 187, in <lambda>
    call = lambda f, *a, **k: f(*a, **k)

  File "/Users/danielsaxton/opt/miniconda3/envs/pandas-dev/lib/python3.8/site-packages/IPython/core/magics/execution.py", line 1114, in timeit
    ast_setup = self.shell.compile.ast_parse(transform(stmt))

  File "/Users/danielsaxton/opt/miniconda3/envs/pandas-dev/lib/python3.8/site-packages/IPython/core/compilerop.py", line 101, in ast_parse
    return compile(source, filename, symbol, self.flags | PyCF_ONLY_AST, 1)

  File "<unknown>", line 1
    sum(
        ^
SyntaxError: unexpected EOF while parsing
```